src/main/java/org/mozilla/jss/JSSProvider.java

```diff
@@ -88,6 +102,22 @@ public CryptoManager getCryptoManager() {
         return cm;
     }
 
+    @Override
+    public Service getService(String type, String algorithm) {
+        if (isAlgorithmDeprecated(algorithm)) {
+            logger.warn("Use of the {} algorithm is deprecated, consider using a more secure algortihm", algorithm);
```

Comment:
There's actually a typo: `algortihm`.

Reply:
Thanks, I actually tried to make sure that I had spelt it right as algorithm is one of my blind-spot words. No matter how many times I write it I still have to check I have spelt it correctly!

Acquisition is another I always have to check, which was unfortunate in my previous employment in the Data Acquisition Group :rofl: